package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@openzeppelin/defender-deploy-client-cli",
-  "version": "0.0.1-alpha.4",
```

Comment:
Since Defender is no longer in beta, we should plan to transition this component to a semver scheme at some point. But since we are still planning breaking changes in this specific component, I agree we can keep this as `-alpha` for now.

Reply:
> Since Defender is no longer in beta

Right, I forgot this. Let's plan the final release but not rush it.